content/modules/ROOT/pages/module-00.adoc

```diff
@@ -89,36 +89,55 @@ Think of Mattermost as a stand-in for ServiceNow in this demonstration.
 	1.	Open mattermost using the above details
 	2.	Click on “View in Browser” to open the console.
 	3.	Login using the credentials provided.
+  4.  Click on the team "Automators"
 
 Once logged in, click on the Town Square channel from the left-hand navigation.
 
 image::mattermost_town_square_placeholder.png[Click on Town Square, 60%, link=“replace_with_actual_image_later”]
 
 === What You’ll See in Town Square
 
-In the Town Square channel, you will begin seeing:
+In the Town Square channel, you will eventually see:
+
 	•	🔍 Error logs from the Apache HTTPD service (collected by Filebeat and forwarded via Kafka)
 	•	🧠 AI Insights generated from the logs using RHEL AI
 
 This real-time feed mimics how production environments might use automated ticket enrichment — by capturing logs and insights and sending them directly to a ticketing system like ServiceNow.
 
+[quote]
+You won’t see any messages just yet — this step is simply to explain where Mattermost fits in and why this channel matters. Logs and insights will start appearing once the first workflow runs.
+
 [quote]
 💡 With this integration, you are essentially watching AI and automation work together — detecting the issue, diagnosing it, and preparing remediation guidance before anyone intervenes.
 
 
 == Build the Workflow
 
-Login to Ansible Automation Platform.
+[cols="2,2,2", options="header"]
+|===
+| System | URL | Credentials
+
+| Ansible Automation Platform
+| AAP is preloaded in the lab interface.
+Click link if you want to open it in full tab:
+https://{nginx_web_url}[AAP Web UI,window=_blank]
+| Username: `{lab_username}`
+Password: `{ssh_password}`
```

Comment:
Check this. Right now, it will just render as a single line and it seems like you want it on two separate lines